src/dev/update_check.ts

```diff
@@ -36,6 +36,20 @@ function getFreshCacheDir(): string | null {
   return null;
 }
 
+const SEMVER_REG = /^(\d+)\.(\d+)\.(\d+)$/;
+function sortSemver(current: string, latest: string): number {
```

Comment:
Perhaps, you could instead use https://deno.land/std@0.195.0/semver/gt.ts?s=gt

Reply:
Oh nice! Didn't know that we have a semver library in our standard library